app/next-client-app/components/AnimatedBeamMapping.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+
+import React, { forwardRef, useRef } from "react";
+
+import { cn } from "@/lib/utils";
+import { AnimatedBeam } from "@/components/magicui/animated-beam";
+import Image from "next/image";
+import { Rabbit } from "lucide-react";
+
+export const Circle = forwardRef<
+  HTMLDivElement,
+  { className?: string; children?: React.ReactNode }
+>(({ className, children }, ref) => {
+  return (
+    <div
+      ref={ref}
+      className={cn(
+        "z-10 flex flex-col lg:size-24 size-20 items-center dark:text-black justify-center rounded-xl border-2 border-border bg-white p-3 shadow-[0_0_20px_-12px_rgba(0,0,0,0.8)]",
+        className
+      )}
+    >
+      {children}
+    </div>
+  );
+});
+
+Circle.displayName = "Circle";
+
+export function AnimatedBeamMapping({ className }: { className?: string }) {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const div1Ref = useRef<HTMLDivElement>(null);
+  const div2Ref = useRef<HTMLDivElement>(null);
+  const div3Ref = useRef<HTMLDivElement>(null);
+  const div6Ref = useRef<HTMLDivElement>(null);
+  const div7Ref = useRef<HTMLDivElement>(null);
+  const div8Ref = useRef<HTMLDivElement>(null);
+  const div9Ref = useRef<HTMLDivElement>(null);
+
+  return (
+    <div
+      className={cn(
+        "relative flex items-center justify-center lg:p-8 mt-5",
+        className
+      )}
+      ref={containerRef}
+    >
+      <div className="flex lg:size-full flex-row lg:items-stretch justify-between gap-10 lg:max-w-lg">
+        <div className="flex flex-col justify-center gap-2">
+          <Circle ref={div1Ref}>
+            <Rabbit className="mr-2" /> Metadata
+          </Circle>
+          <Circle ref={div2Ref}>
+            <h2>Cough</h2>
+          </Circle>
+          <Circle ref={div3Ref}>
```

Comment:
I wonder if this might look better with fewer components - could you show me it without the `Male` Mapping?

Reply:
<img width="1329" alt="Screenshot 2024-09-05 at 12 51 26" src="https://github.com/user-attachments/assets/87330f7f-6d58-472c-b04d-aa272c8d47b6">
<img width="1329" alt="Screenshot 2024-09-05 at 12 51 31" src="https://github.com/user-attachments/assets/2392487a-f2fe-4ff8-be6c-681b8d6a97a3">

I think it will be less balanced, maybe